8.8/package-vipsdev.sh

```diff
@@ -15,59 +15,70 @@ fi
 
 deps="${1:-all}"
 
-echo copying install area $installdir
-
+# Make sure that the repackaging dir is empty
 rm -rf $repackagedir
-cp -r $installdir $repackagedir
+mkdir -p $repackagedir/bin
 
-echo generating import files 
+echo "Copying libvips and dependencies"
 
-./gendeflibs.sh
+# Copy libvips and dependencies with pe-util
+/usr/local/bin/peldd \
+  $installdir/bin/libvips-cpp-42.dll \
+  --clear-path \
+  --path $installdir/bin \
+  --path /usr/lib/gcc/x86_64-w64-mingw32/*-posix \
+  --path /usr/x86_64-w64-mingw32/lib/ \
+  -a \
+  -w USERENV.dll \
+  -w USP10.dll \
+  -w DNSAPI.dll \
+  -w IPHLPAPI.DLL \
+  -w MSIMG32.DLL | xargs cp -t $repackagedir/bin
 
-echo cleaning build $repackagedir
+echo "Copying install area $installdir"
 
-( cd $repackagedir ; rm -rf _jhbuild )
+# Follow symlinks when copying /share and /etc
+cp -Lr $installdir/{share,etc} $repackagedir
 
-for i in COPYING ChangeLog README.md AUTHORS; do 
-  ( cp $basedir/$checkoutdir/$vips_package-$vips_version.$vips_minor_version/$i $repackagedir )
-done
+# Copy everything from /lib and /include, then delete the symlinks
+cp -r $installdir/{lib,include} $repackagedir
+find $repackagedir/{lib,include} -type l -exec rm -f {} \;
 
-# clean /bin 
-( cd $repackagedir/bin ; mkdir ../poop ; mv *vips* ../poop ; mv *.dll ../poop ; rm -f * ; mv ../poop/* . ; rmdir ../poop )
+echo "Generating import files"
+
+./gendeflibs.sh
 
-( cd $repackagedir/bin ; rm -f vips-8.* )
+echo "Cleaning unnecessary files / directories"
 
-( cd $repackagedir/bin ; strip --strip-unneeded *.exe )
+# TODO Do we need to keep /share/doc and /share/gtk-doc?
+rm -rf $repackagedir/share/{aclocal,bash-completion,doc,glib-2.0,gtk-2.0,info,gtk-doc,installed-tests,man,thumbnailers,xml}
```

Comment:
I've removed the `gtk-doc` and `doc` directories. Most people will look for the manuals online, I think.